week3/body_mass.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/python
```

Comment:
Good for thinking about the shebang, however I think that even if this is working on your computer, the better way is to use:
`#!/usr/bin/env python3`
to indicate that you are using python3 (even if `#!/usr/bin/env python` might work as well).
Have a look a this thread: http://stackoverflow.com/questions/6908143/should-i-put-shebang-in-python-scripts-and-what-form-should-it-take